[FEATURE] Update tmp-ia-fonctionnement-debut.json

JTEPIX opened the pull request:
Modifications du module suite aux retours de Sophie et Françoise

## 🔆 Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## ⛱️ Proposition

<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## 🌊 Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🏄 Pour tester
https://app-pr13109.review.pix.fr/modules/tmp-ia-fonctionnement-debut/details

JTEPIX committed:
Update tmp-ia-fonctionnement-debut.json

Modification du module suite aux retours de Sophie et Françoise

pix-bot-github commented:
Choisir les applications à déployer :

- [x] [API](https://api-pr13109.review.pix.fr/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr13109/environment) <!-- pix-api-review -->
- [x] [App (.fr)](https://app-pr13109.review.pix.fr) – [App (.org)](https://app-pr13109.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-app-review-pr13109/environment) <!-- pix-app-review -->
- [ ] [Orga (.fr)](https://orga-pr13109.review.pix.fr) – [Orga (.org)](https://orga-pr13109.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-orga-review-pr13109/environment) <!-- pix-orga-review -->
- [ ] [Certif (.fr)](https://certif-pr13109.review.pix.fr) – [Certif (.org)](https://certif-pr13109.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-certif-review-pr13109/environment) <!-- pix-certif-review -->
- [ ] [Junior](https://junior-pr13109.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-junior-review-pr13109/environment) <!-- pix-junior-review -->
- [ ] [Admin](https://admin-pr13109.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-admin-review-pr13109/environment) <!-- pix-admin-review -->
- [ ] [API MaDDo](https://pix-api-maddo-review-pr13109.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr13109/environment) <!-- pix-api-maddo-review -->
- [ ] [Audit Logger](https://pix-audit-logger-review-pr13109.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr13109/environment) <!-- pix-audit-logger-review -->

> [!IMPORTANT]
> N'oubliez pas de déployer l'API pour pouvoir accéder aux fronts et/ou à l’API MaDDo.

JTEPIX committed:
Update tmp-ia-fonctionnement-debut.json

suppression des doublons

asilnomis committed:
Update tmp-ia-fonctionnement-debut.json

JTEPIX committed:
Merge branch 'dev' into temp-modif-module-ia-novice-fct

JTEPIX committed:
Update tmp-ia-fonctionnement-debut.json

modification du POI pour ne pas avoir le dernier écran et pour pouvoir poser la question (suite au retour SPS et FTO)

JTEPIX committed:
Merge branch 'dev' into temp-modif-module-ia-novice-fct

JTEPIX committed:
Merge branch 'dev' into temp-modif-module-ia-novice-fct

pix-service commented:
:tada: This PR is included in version 5.181.0 :tada:

The release is available on [GitHub release](https://github.com/1024pix/pix/releases/tag/v5.181.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
